src/schema.jl

```diff
@@ -53,6 +53,19 @@ Base.merge!(a::Schema, b::Schema) = (merge!(a.schema, b.schema); a)
 Base.keys(schema::Schema) = keys(schema.schema)
 Base.haskey(schema::Schema, key) = haskey(schema.schema, key)
 
+function Base.:(==)(first::Schema, second::Schema)
+    first === second && return true
+    first.schema === second.schema && return true
```

Comment:
Note that this will be wrong if the dict contains `missing` (recursively). Can this happen?

Reply:
As a key or a value?  Not possible either way at the moment (unless it's a pathological manuallly constructed instance)

Reply:
As a value I think, as for keys dicts use `isequal`.